ffi/dotnet/Devolutions.Picky/Generated/Argon2.cs

```diff
@@ -45,9 +45,7 @@ public static Argon2 New(Argon2Algorithm algorithm, Argon2Params parameters)
             {
                 throw new ObjectDisposedException("Argon2Params");
             }
-            IntPtr resultPtr = Raw.Argon2.New(algorithmRaw, parametersRaw);
-            Raw.Argon2FfiResultBoxArgon2BoxPickyError result = Marshal.PtrToStructure<Raw.Argon2FfiResultBoxArgon2BoxPickyError>(resultPtr);
-            Raw.Argon2FfiResultBoxArgon2BoxPickyError.Destroy(resultPtr);
+            Raw.Argon2FfiResultBoxArgon2BoxPickyError result = Raw.Argon2.New(algorithmRaw, parametersRaw);
```

Comment:
issue (blocking): It seems you generated the bindings using the wrong version of diplomat. Can you make sure you installed our fork instead? Instructions here: https://github.com/Devolutions/picky-rs/tree/master/ffi#justfile

Reply:
having problem install your fork as well, looks like some dependencies needs newer version of rust
```
cargo install --git https://github.com/CBenoit/diplomat.git --rev f4d93d57e93025c57e75e9711d2418faa9311a42 diplomat-tool -f
    Updating git repository `https://github.com/CBenoit/diplomat.git`
  Installing diplomat-tool v0.5.2 (https://github.com/CBenoit/diplomat.git?rev=f4d93d57e93025c57e75e9711d2418faa9311a42#f4d93d57)
    Updating crates.io index
  Downloaded proc-macro2 v1.0.79
  Downloaded basic-toml v0.1.9
  Downloaded syn v2.0.53
  Downloaded 3 crates (352.3 KB) in 0.92s
error: failed to compile `diplomat-tool v0.5.2 (https://github.com/CBenoit/diplomat.git?rev=f4d93d57e93025c57e75e9711d2418faa9311a42#f4d93d57)`, intermediate artifacts can be found at `C:\Users\jou\AppData\Local\Temp\cargo-installW8lLQO`

Caused by:
  package `colored v2.1.0` cannot be built because it requires rustc 1.70 or newer, while the currently active rustc version is 1.67.1
  Either upgrade to rustc 1.70 or newer, or use
  cargo update -p colored@2.1.0 --precise ver
  where `ver` is the latest version of `colored` supporting rustc 1.67.1
error: Recipe `diplomat-install` failed on line 25 with exit code 101
PS C:\Users\jou\code\picky-rs\ffi\dotnet> rustc --version
rustc 1.67.1 (d5a82bbd2 2023-02-07)
```

Reply:
should I update the toolchain version of picky-rs?

Reply:
> will we be using this version for IronRDP as well?

No, we should switch to the latest

Reply:
> should I update the toolchain version of picky-rs?

You could but in a separate PR, please. This will bring other unrelated changes such as fixes for the new clippy lints that I don’t want in this PR. You can temporarily update the file locally just for the purpose of compiling diplomat-tool.

Reply:
Separate PR here https://github.com/Devolutions/picky-rs/pull/258